uc8151/uc8151.go

```diff
@@ -201,6 +201,31 @@ func (d *Device) DrawBitmap(x, y int16, bitmap pixel.Image[pixel.Monochrome]) er
 	return nil
 }
 
+// DrawBuffer copies the buffer directly to the screen at the given coordinates (no rotation taken into account). X coordinate and width need to be a multiple of 8.
```

Comment:
> (no rotation taken into account)

That seems is a possible issue, since most of the time with this screen we operate at 270 degrees rotation.